src/ui_basic/dropdown.h

```diff
@@ -21,8 +21,10 @@
 #define WL_UI_BASIC_DROPDOWN_H
 
 #include <deque>
+#include <graphic/image_cache.h>
```

Comment:
Always use `""` for Widelands includes and put it them the last include block. I'm surprised codecheck doesn't complain…